src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,29 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    enum Color {
+        RED,
+        GREEN,
+        BLUE,
+        YELLOW,
+        ORANGE
+    }
+
+    private final int index = new Random().nextInt(Color.values().length);
+    private final Color color = Color.values()[getIndex()];
+
+    public int getIndex() {
+        return index;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
     public String getRandomColor() {
-        return null;
+        return color.toString();
```

Comment:
see correct method in checklist
```suggestion
        return color.????();
```